### ISSUE TITLE
5. Plot graphs

### ISSUE THREAD

**Issue by chovancova** [chovancova/ML-Project-Artistic-Style-Transfer#5]
Plot graphs showing how your training and validation loss and accuracy changed with respect to the epochs completed.
